### PR TITLE
Fix broken image links in DevTools release notes

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This will fix https://github.com/flutter/devtools/issues/5767.

### CODE REVIEW

File: src/tools/devtools/release-notes/release-notes-2.23.1-src.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 * Use the default Flutter service worker. [#5331](https://github.com/flutter/devtools/pull/5331)
 * Added the new verbose logging feature for helping us debug user issues. [#5404](https://github.com/flutter/devtools/pull/5404)
 
-![verbose logging](images-2.23.1/verbose-logging.png "verbose_logging")
+![verbose logging]({{site.url}}/development/tools/devtools/release-notes/images-2.23.1/verbose-logging.png "verbose_logging")
```
Comment (parlough) [inline src/tools/devtools/release-notes/release-notes-2.23.1-src.md:10]:
```suggestion
![verbose logging]({{site.url}}/tools/devtools/release-notes/images-2.23.1/verbose-logging.png "verbose_logging")
```

File: src/tools/devtools/release-notes/release-notes-2.23.1-src.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 ## CPU profiler updates
 * Add a Method Table to the CPU profiler. [#5366](https://github.com/flutter/devtools/pull/5366)
 
-![method table](images-2.23.1/cpu-method-table.png "method_table")
+![method table]({{site.url}}/development/tools/devtools/release-notes/images-2.23.1/cpu-method-table.png "method_table")
```
Comment (parlough) [inline src/tools/devtools/release-notes/release-notes-2.23.1-src.md:26]:
```suggestion
![method table]({{site.url}}/tools/devtools/release-notes/images-2.23.1/cpu-method-table.png "method_table")
```

**Review by parlough — COMMENTED** [review]
Thanks for catching and fixing these.

We've broken up the `/development` directory, so have to adjust these a bit:

We'll have to wait until after the freeze ends soon to merge.

**Comment by kenzieschmoll** [comment]
Why does /development need to be added? I don't see `development` in the path to these images and src files? https://github.com/flutter/website/tree/main/src/tools/devtools/release-notes

If that is the case, all the links in the previous DevTools release notes will be broken too since they don't reference `development`. However, the previous release notes images don't seem broken on the website: https://docs.flutter.dev/tools/devtools/release-notes/release-notes-2.20.0

**Comment by parlough** [comment]
`/development` doesn't need to be added, in those review suggestions I'm removing it.

**Comment by kenzieschmoll** [comment]
Oh whoops, I must have been on an old branch when I copied the file path. Fixed

**Review by parlough — APPROVED** [review]
Thanks!

**Pull request merged**